main/src/main/resources/application.conf

```diff
@@ -20,7 +20,7 @@ outDir = ${rootDir}/output
 
 # this is the directory that stores the raw nxml, .csv, and/or .tsv files
 # this directory *must* exist
-papersDir = ${rootDir}/papers
+papersDir = ${rootDir}/papers/corpus_20200513_control_1  # change this directory for debugging
```

Comment:
You might want to change this back to the default before merging

Reply:
Oh sorry, I will change this to the default after I fix this.